src/langchain/index.ts

```diff
@@ -14,13 +14,10 @@ import { AUTOBLOCKS_INGESTION_KEY, readEnv } from '../util';
 export class AutoblocksCallbackHandler extends BaseCallbackHandler {
   name = 'autoblocks_handler';
 
-  private tracer: AutoblocksTracer;
+  private _tracer: AutoblocksTracer;
   private traceId: string | undefined = undefined;
 
-  constructor(args?: {
-    traceId?: string;
-    properties?: Record<string, unknown>;
```

Comment:
dropping these constructor args since you can just set them via the tracer like `handler.tracer.setTraceId` or `handler.tracer.updateProperties`